pkg/util/v1alpha3/katibclient/katib_client.go

```diff
@@ -41,7 +41,6 @@ type Client interface {
 	GetTrialList(name string, namespace ...string) (*trialsv1alpha3.TrialList, error)
 	GetTrialTemplates(namespace ...string) (map[string]string, error)
 	UpdateTrialTemplates(newTrialTemplates map[string]string, namespace ...string) error
-	GetMetricsCollectorTemplates(namespace ...string) (map[string]string, error)
 	UpdateMetricsCollectorTemplates(newMCTemplates map[string]string, namespace ...string) error
```

Comment:
Is UpdateMetricsCollectorTemplates  needed?

Reply:
delete it, too
Thanks